src/Database/InfluxDB/Line.hs

```diff
@@ -29,6 +32,7 @@ import qualified Data.ByteString.Lazy as L
 import qualified Data.Map.Strict as Map
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as TE
+import GHC.Generics
```

Comment:
nitpick: Could you move this import to the end of the previous section? I organize imports in this order: modules from base, libraries, and the internal library. Each group is sorted alphabetically and qualified imports come later.